cmd/kubeadm/app/cmd/token.go

```diff
@@ -53,25 +53,24 @@ func NewCmdToken(out io.Writer, errW io.Writer) *cobra.Command {
 	var dryRun bool
 	tokenCmd := &cobra.Command{
 		Use:   "token",
-		Short: "Manage bootstrap tokens.",
+		Short: "Manage Bootstrap Tokens.",
 		Long: dedent.Dedent(`
-			This command will manage Bootstrap Token for you.
-			  Please note this usage of this command is optional, and mostly for advanced users.
+			This command will manage a Bootstrap Token for you.
+			Please note that the usage of this command is optional and for advanced users.
 
 			In short, Bootstrap Tokens are used for establishing bidirectional trust between a client and a server.
-			A Bootstrap Token can be used when a client (for example a node that's about to join the cluster) needs
+			A Bootstrap Token can be used when a client (for example a node that is about to join the cluster) needs
```

Comment:
Nit: Maybe we can drop the weird title casing on "Bootstrap Token" in this section and just say "bootstrap token"? I don't think it necessarily needs to be capitalized and it scans weird for me.

Reply:
@mattmoyer 
yeah, the capitalization of "Boostrap Token(s)" was consistent in a number of places. should i just make it lower case (i.e. "bootsrap token(s)") everywhere?